This patch corrects the partition labels, sizes, and offsets, to match
those that are used by u-boot and userspace.

diff --git a/arch/arm/boot/dts/sun5i-r8-chip.dts b/arch/arm/boot/dts/sun5i-r8-chip.dts
index f894cb50..21fe5a3 100644
--- a/arch/arm/boot/dts/sun5i-r8-chip.dts
+++ b/arch/arm/boot/dts/sun5i-r8-chip.dts
@@ -186,17 +186,17 @@
 		nand-ecc-mode = "hw";
 
 		spl@0 {
-			label = "SPL";
+			label = "spl";
 			reg = /bits/ 64 <0x0 0x400000>;
 		};
 
 		spl-backup@400000 {
-			label = "SPL.backup";
+			label = "spl-backup";
 			reg = /bits/ 64 <0x400000 0x400000>;
 		};
 
 		u-boot@800000 {
-			label = "U-Boot";
+			label = "uboot";
 			reg = /bits/ 64 <0x800000 0x400000>;
 		};
 
@@ -205,9 +205,14 @@
 			reg = /bits/ 64 <0xc00000 0x400000>;
 		};
 
-		rootfs@1000000 {
-			label = "rootfs";
-			reg = /bits/ 64 <0x1000000 0x1ff000000>;
+		swap@1000000 {
+			label = "swap";
+			reg = /bits/ 64 <0x1000000 0xC000000>;
+		};
+
+		UBI@1a000000 {
+			label = "UBI";
+			reg = /bits/ 64 <0xD000000 0x1f3000000>;
 		};
 	};
 };
